document changes from PR#3037 (#1844)
OttoWinter/esphomedocs: components/sensor/tcs34725.rst

Configuration variables:
------------------------

> - **red_channel** (*Optional*): Get the percentage of red color channel in relation to the ``clear_channel``.
> - **green_channel** (*Optional*): Get the percentage the green color channel in relation to the ``clear_channel``.
> - **blue_channel** (*Optional*): Get the percentage of the blue color channel in relation to the ``clear_channel``.
> - **clear_channel** (*Optional*): Get the percentage of the clear (without a color filter)
>   channel in relation to the maximum value for the choosen ``integration_time``. 

Reviewer: Let's leave the sensor references in here and clarify that it's a relative value:
```suggestion
- **red_channel** (*Optional*): Value of the red color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **green_channel** (*Optional*): Value of the green color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **blue_channel** (*Optional*): Value of the blue color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **clear_channel** (*Optional*): Vaule of the clear (without a color filter) channel, relative to the maximum value for
  the chosen integration time. All options from :ref:`Sensor <config-sensor>`.
```

  All options from :ref:`Sensor <config-sensor>`.
- **clear_channel** (*Optional*): Get the percentage of how strongly the clear (without a color filter)
  channel is activated. All options from :ref:`Sensor <config-sensor>`.
- **red_channel** (*Optional*): Get the percentage of red color channel in relation to the ``clear_channel``.
- **green_channel** (*Optional*): Get the percentage the green color channel in relation to the ``clear_channel``.
- **blue_channel** (*Optional*): Get the percentage of the blue color channel in relation to the ``clear_channel``.
- **clear_channel** (*Optional*): Get the percentage of the clear (without a color filter)
  channel in relation to the maximum value for the choosen ``integration_time``. 
- **illuminance** (*Optional*): Get the total illuminance of the sensor in lx.

>   All options from :ref:`Sensor <config-sensor>`.

Reviewer: I wouldn't remove this, now it's unclear what you should specify for this setting.

Author: ok

- **color_temperature** (*Optional*): Get the calculated color temperature of the light in Kelvin.

>   All options from :ref:`Sensor <config-sensor>`.

Reviewer: Same.

Configuration variables:
------------------------

- **red_channel** (*Optional*): Value of the red color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **green_channel** (*Optional*): Value of the green color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **blue_channel** (*Optional*): Value of the blue color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.

> - **clear_channel** (*Optional*): Vaule of the clear (without a color filter) channel, relative to the maximum value for

Reviewer 2: ```suggestion
- **clear_channel** (*Optional*): Value of the clear (without a color filter) channel, relative to the maximum value for
```

- **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
  sensor. Defaults to ``60s``.

> Since the raw R+G+B values reported by the sensor are approximately equal to the clear value, ``red_channel``, ``green_channel`` and ``blue_channel`` percentages are calculated in relation to the ``clear_channel``.

Reviewer: Isn't this already explained above?

Author: removed
